plugins/realtek-mst/README.md

```diff
@@ -5,6 +5,9 @@
 This plugin updates the firmware of DisplayPort MST hub devices made by Realtek,
 such as the RTD2141b and RTD2142.
 
+This plugin was neither written by, verified or endorsed by Realtek Semiconductor
+Corp, and so should be used with extreme caution.
```

Comment:
how is this any different than most of the other plugins?  Why is it special to receive such a message in README?

Reply:
Realtek expressed concern.

Reply:
So instead, can't they review the plugin and endorse it?  It feels like it's 2 steps away from person reports "there is a problem with that tool" to realtek and they say "we only support `realtek-mst-flasher.so`".

Reply:
Specifically r8152 comes to mind...

Reply:
I'll poke them offline.

Reply:
The implication that the correctness of the plugin ("extreme caution") depends on Realtek's endorsement seems weird- what about "it is not supported by Realtek," which would also short-circuit @superm1's hypothetical?

Reply:
So `This plugin was neither written by, verified, supported or endorsed by Realtek Semiconductor Corp.`? Or still too wordy?

Reply:
Could remove the first "by", but I'm happy with that.

Reply:
> So instead, can't they review the plugin and endorse it

That'd be a _no_ from them.